### PR TITLE
Make Kårres properly display when there is no lunch

### PR DESCRIPTION
**Pull request by Multipacker**

### CODE REVIEW

File: src/Model/Karen.hs [hunk 1]
```diff
@@ -108,6 +108,7 @@ parse lang =
         >=> mapM menuParser
         )
       )
+    >=> pure . filter ((/= "stängt") . _mFood)
```
Comment (The1Penguin) [inline src/Model/Karen.hs]: Could instead lens for it to be more inline with the rest of the code

File: src/Model/Types.hs [hunk 2]
```diff
@@ -28,5 +28,8 @@ data NoMenu
 
 -- | Menu of a restaurant.
 -- Title, Body text
-data Menu = Menu Text Text
+data Menu = Menu
+  { _mName :: Text
+  , _mFood :: Text
+  }
   deriving (Eq, Show)
```
Comment (The1Penguin) [inline src/Model/Types.hs:37]:
```suggestion
data Menu = Menu
  { _mName :: Text
  , _mFood :: Text
  }
  deriving (Eq, Show)
  
  makeLenses ''Menu
```

would create lenses with the names `mName` and `mFood`.

File: src/Model/Karen.hs (hunk 1 above)
Comment (The1Penguin) [reply src/Model/Karen.hs]: Could also use `filterM` and make the fetch a `pure`

File: src/Model/Karen.hs [hunk 3]
```diff
@@ -108,6 +110,7 @@ parse lang =
         >=> mapM menuParser
         )
       )
+    >=> filterM (((/= "stängt") <$>) <$> (^.^ mFood))
```
Comment (The1Penguin) [inline src/Model/Karen.hs]: [Now this is some good food](https://www.youtube.com/watch?v=bPRn9TA1cDc)

File: src/Util.hs [hunk 4]
```diff
@@ -28,3 +29,7 @@ menusToEitherNoLunch = \case
 removeWhitespaceTags :: [Tag ByteString] -> [Tag ByteString]
 removeWhitespaceTags =
   filter (or . ([not . isTagText, tagText (not . BL.all W8.isSpace)] <*>) . pure)
+
+(^.^) :: Monad m => s -> Getting a s a -> m a
+(^.^)  = (pure .) . (^.)
+infixl 8 ^.^
```
Comment (The1Penguin) [inline src/Util.hs:35]: They are so happy now ^.^

File: src/Model/Types.hs [hunk 5]
```diff
@@ -28,5 +30,10 @@ data NoMenu
...
+
+makeLenses ''Menu
```
Comment (The1Penguin) [inline src/Model/Types.hs:39]: Lenses are good to be inline with the style of code in this repo

**Review by The1Penguin — APPROVED** [review]
LGTM

**Comment by SKyletoft** [comment]
Snälla nej

**Comment by The1Penguin** [comment]
This fixes #130

**Pull request merged**